comp/metadata/packagesigning/utils/apt.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package utils
+
+import (
+	"bufio"
+	"os"
+)
+
+const (
+	packageConfig = "/etc/dpkg/dpkg.cfg"
+)
+
+// IsPackageSigningEnabled returns the signature policy for the host. When no-debsig is written (and uncommented) in the configuration it means GPG package signing verification is disabled
+func IsPackageSigningEnabled() bool {
```

Comment:
philosophical nit: This and a decent number of other functions in the PR tend to hide an error condition from the caller, returning empty strings or nils which is likely a bad idea as the caller should ultimately be responsible for deciding if an error is ok to ignore.

Reply:
Indeed I considered most of the time there was no need to raise specific errors to the top in case of issue while collecting information for this telemetry (I also felt it was making the code more complex to read, but this is maybe expected). I've updated a bunch of places according to your comments